core/templates/dev/head/pages/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -71,21 +71,21 @@
               (<[getChangeListCount()]>)
             </span>
           </span>
-          <span ng-if="!isPrivate" title="Publish Changes">
+          <span ng-if="!collectionRights.isPrivate()" title="Publish Changes">
```

Comment:
Maybe: ``collectionRights.isPublic()``, as that's why we have that method and would make this more readable. :)